cmd/download-plugins/main.go

```diff
@@ -73,15 +73,23 @@ func run() error {
 		if err != nil {
 			return err
 		}
-		if exists {
-			log.Printf("already downloaded: %s", filepath.Join(downloadDir, filepath.Base(pluginRelease.URL)))
-		} else if err := downloadReleaseToDir(ctx, client.GitHub.Client(), pluginRelease, downloadDir); err != nil {
+		if !exists {
+			if err := downloadReleaseToDir(ctx, client.GitHub.Client(), pluginRelease, downloadDir); err != nil {
+				return err
+			}
+		}
+		if err := setPluginZipTimestamp(pluginRelease, downloadDir); err != nil {
```

Comment:
This felt like a nice change to make file's modified time on disk match when they were released.